include/itkBoneMorphometryFeaturesImageFilterPostProcessing.h

```diff
@@ -0,0 +1,100 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkBoneMorphometryFeaturesImageFilterPostProcessing_h
+#define itkBoneMorphometryFeaturesImageFilterPostProcessing_h
+
+#include "itkImageToImageFilter.h"
+#include "itkNumericTraits.h"
+#include "itkSimpleDataObjectDecorator.h"
+
+//useful filters
+#include <itkVectorIndexSelectionCastImageFilter.h>
+#include <itkMinimumMaximumImageFilter.h>
+#include <itkMaskImageFilter.h>
+#include <itkComposeImageFilter.h>
+
+#include <vector>
+
+namespace itk
+{
+/** \class BoneMorphometryFeaturesImageFilterPostProcessing
```

Comment:
A more specific name for the filter would help. For example,

`itk::ReplaceFeatureMapNanInfImageFilter`